docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -73,19 +129,53 @@ the ``copy`` module::
     ...
     '''
 
-Building & Testing
+Formatting options
 ''''''''''''''''''
+These formatting functions are ``U()``, ``M()``, ``I()``, and ``C()``
+for URL, link to help for another module, `italic`, and ``constant-width`` respectively. It is suggested
+to use ``C()`` for file and option names, and ``I()`` when referencing
+parameters; module names should be specified as ``M(module)`` to create a
+link to the online documentation for that module.
+
 
-Put your completed module file into the 'library' directory and then
+Example usage::
+
+    Or if not set the environment variable C(ACME_PASSWORD) will be used.
+    ...
+    Required if I(state=present)
```

Comment:
I suppose with a strict reading of the I() and C() rules and jtyr's proposed wording, this becomes ambiguous since it's a combination.  Might want to call it out specifically or clarify the wording to show that this is the desired state. "For option values or combinations of option name and value, use I()"